src/background_thread.c

```diff
@@ -364,7 +364,11 @@ background_thread_create(tsd_t *tsd, unsigned arena_ind) {
 	pre_reentrancy(tsd);
 	int err;
 	load_pthread_create_fptr();
-	if ((err = pthread_create(&info->thread, NULL,
+	/*
+	 * To avoid complications (besides reentrancy), create internal
```

Comment:
Ah, I think I understand.  This workaround only helps us if we don't have ```JEMALLOC_LAZY_LOCK``` defined, right?